CODEOWNERS

```diff
@@ -65,6 +65,8 @@
 /soc/x86/                                 @andrewboie @jenmwms @aasthagr
 /arch/xtensa/                             @andrewboie @dcpleung @andyross
 /soc/xtensa/                              @andrewboie @dcpleung @andyross
+/arch/sparc/                              @martin-aberg
```

Comment:
I would add yourself to the MAINTAINERS file as well.

Reply:
Sure. I will have a look at this. What would the appropriate role be,  maintainer or collaborator?

Reply:
If you're willing to be the maintainer, that would be ideal. We can get you added to the GH contributors list so you can be added to code review. cc @MaureenHelm

Reply:
We need a maintainer, but please start as a collaborator first. You can find more details in the [Project Roles](https://docs.zephyrproject.org/latest/contribute/project_roles.html) document.

Reply:
I have now added myself as collaborator in `MAINTAINERS.yml`: b4753af07bc54ad7171be8e740beea210b66f345